content/en/Platform Deep Dive/Pentests/Findings/remediate-findings.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Review and Take Action on Findings"
+linkTitle: "Take Action on Findings"
+weight: 20
+description: >
+  Remediate or accept findings.
+---
+
+{{% pageinfo %}}
+During remediation, review findings that our pentesters discovered, and take action on them.
+{{% /pageinfo %}}
+
+Once our pentesters move a finding to [Pending Fix](/platform-deep-dive/pentests/findings/finding-states/), you can:
+
+- Fix the finding and [submit it for retest](#submit-a-finding-for-retest)
+- [Mark the finding as Accepted Risk](#mark-a-finding-as-accepted-risk)
+
+Our pentesters describe findings and provide recommendations on how to fix them. Navigate to the finding page for details.
+
+## Submit a Finding for Retest
+
+Once you've fixed a finding internally, you can submit it for retest.
+
+1. On the **Findings** tab of the pentest page, filter **Pending Fix** findings.<br><br>
+![Filter Pending Fix findings](/deepdive/FilterPendingFixFindings.png "Filter Pending Fix findings")
+1. Select the desired finding.
+1. In the **State** list, select **Ready for Retest**.<br><br>
+![Submit a finding for retest](/deepdive/ReadyForRetestFinding.png "Submit a finding for retest")
+
+The pentester who posted the finding gets notified and retests the issue. When finished, they change the [state](/platform-deep-dive/pentests/findings/finding-states/) to:
+
+- Fixed, if they can't reproduce the issue.
+- Pending Fix, if the issue persists. Read the pentester's comment for details.
+
+### Free Retesting Duration
+
+The timeline for retesting starts after your pentest end date within an active contract. Mark your findings as **Ready for Retest** at least 10 days before your contract ends.
+
+- Free retesting is based on your {{% ptaas-tier %}}:
+  - Standard tier: 6 months
+  - Premium and Enterprise tiers: 12 months
+- Free retesting is only available within an active contract. Your retest end date will be either the duration of your purchased tier or 10 days before your contract end date.
+- When you start a pentest right before your contract expires, you may not qualify for retesting. If you add a new contract, we’ll update your retest end date based on the tier your pentest was planned in.
+
+{{%expand "See an example." %}}
+The Standard contract duration is from January 1, 2022 to December 31, 2022.
```

Comment:
Wait... where did you get info for "The Standard contract duration"? Is there such a thing?

Unless legal has come to us and asked us to add this info, 

I recommend replacing this section. We can keep the link to https://www.cobalt.io/pentest-pricing. I might even include:

"Details and deadlines for free retesting depends on your contract. For more information, consult your CSM (if you have one), or contact support@cobalt.io."

Reply:
@mjang-cobalt Please see this comment: https://github.com/cobalthq/cobalt-product-public-docs/pull/254#issuecomment-1347425095